release-notes/schema/dotnet-required-packages.json

```diff
@@ -0,0 +1,170 @@
+{
+  "description": "The set of packages required by a given .NET version for a set of distros.",
+  "type": [
+    "object",
+    "null"
+  ],
+  "properties": {
+    "ChannelVersion": {
+      "type": "string"
+    },
+    "EolDate": {
+      "type": "string",
+      "format": "date"
+    },
+    "Packages": {
+      "type": "array",
+      "items": {
+        "description": "A logical package that will exist in various distros with different package names. Includes the scenarios for which the package is required.",
+        "type": [
+          "object",
+          "null"
+        ],
+        "properties": {
+          "Id": {
+            "type": "string"
+          },
+          "Name": {
+            "type": "string"
+          },
+          "References": {
+            "type": [
+              "array",
+              "null"
+            ],
+            "items": {
+              "type": [
+                "string",
```

Comment:
The set of values used here should be constrained. I was thinking this would be an enum type.

Reply:
`References` is intended to be markdown strings, as you can see in `linux-packages.json`. What would that look like?

Reply:
Sorry, I meant this to be under `Required`. I added my comment there instead: https://github.com/dotnet/core/pull/9412/files/03af9b669257a2bc4603bb95c5a7e8f0dee2f727#r1682969587.